src/main/java/com/dawn/jat/illuminati/post/exception/ErrorResponse.java

```diff
@@ -0,0 +1,11 @@
+package com.dawn.jat.illuminati.post.exception;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ErrorResponse {
+    private String errorCode;
```

Comment:
We can use a Map<String,String> for this I think :D

Reply:
private Map<String,String> error;

Reply:
Updated